ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -291,6 +295,27 @@
     public static final String REQUEST_COST = "x-ambry-request-cost";
   }
 
+  /**
+   * Headers with information about user quota and usage.
+   */
+  public static final class UserQuotaHeaders {
```

Comment:
This might be a minor, but I think we will appreciate it later. Can we use the same term on quota here? I remember we are referring this as "RequestQuota" here and there in the code. Can we change this to RequestQuotaHeaders?